src/oidc/oidc.ts

```diff
@@ -270,6 +270,10 @@ export const createUserManager = async (options: CreateUserManagerOptions) => {
             scope: 'openid',
             stateStore: new WebStorageStateStore({ store: window.localStorage }),
             post_logout_redirect_uri: _postLogoutRedirectUri,
+            // this is enabled by default, it runs a silent renew service in the background which triggers the prompt=none auth calls
+            // Source: https://github.com/authts/oidc-client-ts/blob/9ccae8f87b3e9e2df349aaf6f007964ced287b02/src/UserManagerSettings.ts#L140
+            // Notable issue: https://github.com/authts/oidc-client-ts/issues/1174
+            automaticSilentRenew: false,
```

Comment:
<img width="725" alt="Screenshot 2025-02-08 at 3 45 53 PM" src="https://github.com/user-attachments/assets/61ce25d9-3cb0-4374-b799-bfb3b4800436" />

Reply:
We only need the access tokens during the `Callback` page process, where we fetch the legacy tokens. After that, access tokens shouldn't be necessary anymore, and by BE's documentation, the access tokens expires within a minute
<img width="808" alt="Screenshot 2025-02-08 at 4 45 20 PM" src="https://github.com/user-attachments/assets/1d6f5d70-7c0c-4032-9fb9-a9a7e1fadeb1" />